cms/templates/maintenance/_announcement_edit.html

```diff
@@ -36,13 +36,15 @@ <h3 class="info-course">
   <script type="text/javascript">
       tinymce.init({
           selector: "#id_content",
+          height: 450,
+          skin: "studio-tmce5",
           branding: false,
           plugins: [
               "advlist autolink lists link image charmap print anchor",
               "searchreplace visualblocks code",
               "insertdatetime media table contextmenu paste"
           ],
-          toolbar: "undo redo | styleselect | bold italic | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image"
+          toolbar: "undo redo styleselect bold italic alignleft aligncenter alignright alignjustify bullist numlist outdent indent link image"
```

Comment:
Why change this behavior?

Reply:
@leangseu-edx, please see [this comment](https://github.com/openedx/edx-platform/pull/30335#discussion_r862248651).